contrib/systemd/traefik.service

```diff
@@ -1,11 +1,34 @@
 [Unit]
 Description=Traefik
+Documentation=https://docs.traefik.io
+After=network-online.target
+AssertFileIsExecutable=/usr/bin/traefik
+AssertPathExists=/etc/traefik.toml
 
 [Service]
+# Run traefik as its own user (create new user with: useradd -r -s /bin/false -U -M traefik)
+#User=traefik
+#AmbientCapabilities=CAP_NET_BIND_SERVICE
+
+# configure service behavior
 Type=notify
 ExecStart=/usr/bin/traefik --configFile=/etc/traefik.toml
 Restart=always
 WatchdogSec=1s
 
+# lock down system access
+ProtectSystem=strict
+PrivateTmp=true
+ProtectHome=true
+PrivateDevices=true
+ProtectKernelTunables=true
+ProtectControlGroups=true
+
+# allow writing of acme.json
+ReadWritePaths=/etc/acme.json
```

Comment:
You allow only writing `acme.json` but it is not the only one file that traefik can write or read:
- [Logs](https://docs.traefik.io/configuration/logs/)
- [Entrypoints](https://docs.traefik.io/configuration/entrypoints/)

Reply:
The file is only supposed to serve as a general baseline, but I added a comment to the file about the log / entrypoint config.